build.sbt

```diff
@@ -44,51 +44,30 @@ libraryDependencies ++= Seq(
 //  "org.broadinstitute" %% "cromwell" % "0.9" excludeAll ExclusionRule(organization = "com.gettyimages"),
   "org.broadinstitute.dsde.vault" %% "vault-common" % "0.1-15-bf74315",
   "org.mongodb" %% "casbah" % "2.8.2",
-  "org.flywaydb" % "flyway-core" % "3.2.1",
+  "org.flywaydb" % "flyway-core" % "4.0.3",
```

Comment:
you can leave this out of build.sbt too; it'll get pulled in as a dependency from the flyway in plugins.sbt